igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisService.groovy

```diff
@@ -66,20 +67,30 @@ class TravisService implements BuildService {
 
     @Override
     List<GenericGitRevision> getGenericGitRevisions(String inputRepoSlug, int buildNumber) {
-        String repoSlug = cleanRepoSlug(inputRepoSlug)
-        Builds builds = getBuilds(repoSlug, buildNumber)
-        if (builds.commits?.branch) {
-            return builds.commits*.genericGitRevision()
-        }
-        return null
+        new SimpleHystrixCommand<List<GenericGitRevision>>(
+            groupKey, buildCommandKey("getGenericGitRevisions"),
+            {
+                String repoSlug = cleanRepoSlug(inputRepoSlug)
+                Builds builds = getBuilds(repoSlug, buildNumber)
+                if (builds.commits?.branch) {
```

Comment:
`return builds.commits?.branch ? builds.commits*.genericGitRevision() : null` is a bit neater IMO.